scenario_player/environment/development.json

```diff
@@ -1,5 +1,6 @@
 {
     "environment_type": "development",
+    "development_environment": "unstable",
```

Comment:
Should we define this in the other files as well, just for consistency?

Reply:
Since the default choice is the right one for all environments but this one, I prefer to leave it out. It could just cause confusion with the similarly named "environment_type".